README-dev.md

```diff
@@ -10,7 +10,7 @@ Mina, this is the right file!
 
 ## Building Mina
 
-Building Mina is involved because many C library dependencies must be present in the system. OCaml-specific setup is also required.
+Building Mina is involved because many C library dependencies must be present in the system. Furthermore, these libraries need to be in correct versions, or else the system will fail to build. OCaml-specific setup is also required. Therefore, it is recommended to build Mina with Nix, which offers a great help in managing these dependencies. Manual dependency management is fragile and prone to break with every system update.
```

Comment:
```suggestion
Building Mina is involved because many C library dependencies must be present in the system. Furthermore, these libraries must be in the correct versions, or the system will fail to build. OCaml-specific setup is also required. We recommend building Mina with Nix, a functional package manager. Nix ensures that package dependency specifications are complete. Manual dependency management is fragile and prone to break with every system update.
```

Reply:
@Sventimir 
hmmm we can probably omit  "Manual dependency management is fragile and prone to break with every system update."

Reply:
I find it extremely displeasing when others tell me what to do and so I try to avoid doing it myself. Instead I prefer to advise and recommend things, explaining why I think they're better to their alternatives, but ultimately leaving the decision to a well-informed user. I understand that this style complicates the message, though, while people often prefer simplicity over having very detailed information. Indeed, they often try to minimise decision-making when unnecessary.

On another note, I believe it's good to keep documentation impersonal wherever possible. It gives the reader the feeling they read about facts rather than oppinions of a group of people. It's somewhat related to the opinion I expressed above about presenting facts rather than opinions and letting people decide for themselves. Furthermore, it's usually unclear, who exactly that group expressiong their opinions in the documentation is (unless directly specified).

That said, I believe consistency is more important than any of these considerations, so if you think that simplified and/or personal style is better, I'm happy to accept that.

Reply:
@Sventimir  thank you for your comment, I applaud your passion and interest. 
There are many considerations (and different opinions) for technical content, especially for developer docs. 
In this case, the reader of these README docs is the developer, so using second-person pronoun "you" is an established convention that we use for the Mina docs to write for our developer. 
Did you know we have a technical content style guide? https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide#language-and-grammar 
You can always ask questions in the #mf-o1-docs Slack channel, too, so we can elevate our collective wisdom regarding docs (I love docs!)

Reply:
Thank you. I accept your proposals then.